api/src/user.rs

```diff
@@ -1,14 +1,44 @@
-use actix_web::{HttpResponse, Responder, route};
+use actix_web::{HttpResponse, Responder, route, web};
 use build_time::build_time_utc;
 
 pub fn config(cfg: &mut actix_web::web::ServiceConfig) {
-    cfg.service(test);
+    cfg
+    .service(test)
+    .service(delete_user)
+    .service(delete_device)
+    .service(add_device);
 }
 
-#[route("/test", method = "GET", method = "HEAD")]
+#[route("/test", method = "GET")]
```

Comment:
It doesn't really matter as far as I know, but why did you remove the HEAD method?